bigtop-deploy/puppet/modules/hadoop/manifests/init.pp

```diff
@@ -203,7 +203,7 @@
       $hdfs_support_append = undef,
       $hdfs_replace_datanode_on_failure = undef,
       $hdfs_webhdfs_enabled = "true",
-      $hdfs_replication = undef,
+      $hdfs_replication = "1",
```

Comment:
Why changed to 1?

Reply:
The configuration for hdfs and mapred is referred by [Giraph quick start
](http://giraph.apache.org/quick_start.html).